src/utils/blanket.js

```diff
@@ -0,0 +1,11 @@
+import search from "./search";
```

Comment:
This file can be removed, yeah?

Reply:
Yeah, I think this was leftover from a refactor. Will remove it now